### PR TITLE
Aligning structure with our Phoenix business domain conventions

### LINKED ISSUE #38: Aligning structure with our Phoenix business domain conventions
## Why
Currently  the code/folder structure is aligned with our Phoenix [business domain conventions](https://nimblehq.co/compass/development/code-conventions/elixir-phoenix/#business-domain) 

Need to clean up by doing the following actions

-  Move all keyword related functions to the `keywords` context
- Align account/user code and its folder/filename with our [business domain conventions](https://nimblehq.co/compass/development/code-conventions/elixir-phoenix/#business-domain) 

## Who Benefits?
Elixir Developers

### PR DESCRIPTION
**Pull request by junan**
https://github.com/junan/elixir_google_scraper/projects/1#card-63936032
## What happened

Use the `context` for related schema functions and move schemas into `schemas` folder
 
## Insight

- Move `user.ex` to `schemas` folder.
- Move `user_token.ex` to `schemas` folder.
- Introduced `users.ex` context
- Introduced `user_tokens.ex`
- Introduced `keywords` context
 
## Proof Of Work

Spec should be passed

### CODE REVIEW

File: lib/elixir_google_scraper/account/user_token.ex [hunk 1]
```diff
@@ -1,4 +1,4 @@
-defmodule ElixirGoogleScraper.Accounts.UserToken do
+defmodule ElixirGoogleScraper.Account.UserToken do
```
Comment (rosle) [inline lib/elixir_google_scraper/account/user_token.ex]: Should this be schema 🤔

File: lib/elixir_google_scraper/account/user_token.ex [hunk 2]
```diff
@@ -30,7 +30,7 @@ defmodule ElixirGoogleScraper.Accounts.UserToken do
     token = :crypto.strong_rand_bytes(@rand_size)
 
     {token,
-     %ElixirGoogleScraper.Accounts.UserToken{token: token, context: "session", user_id: user.id}}
+     %ElixirGoogleScraper.Account.UserToken{token: token, context: "session", user_id: user.id}}
```
Comment (rosle) [inline lib/elixir_google_scraper/account/user_token.ex]: Is this the current module? If it is, we can use `__MODULE__` 👍

File: lib/elixir_google_scraper/accounts.ex [hunk 3]
```diff
@@ -5,7 +5,8 @@ defmodule ElixirGoogleScraper.Accounts do
```
Comment (rosle) [inline lib/elixir_google_scraper/accounts.ex]: Do we also need to refactor this module as well? 🤔

File: lib/elixir_google_scraper/account/schemas/user.ex [hunk 4]
```diff
@@ -116,7 +116,7 @@ defmodule ElixirGoogleScraper.Accounts.User do
   `Bcrypt.no_user_verify/0` to avoid timing attacks.
   """
   def valid_password?(
-        %ElixirGoogleScraper.Accounts.User{hashed_password: hashed_password},
+        %ElixirGoogleScraper.Account.Schemas.User{hashed_password: hashed_password},
```
Comment (rosle) [inline lib/elixir_google_scraper/account/schemas/user.ex]:
```suggestion
        %__MODULE__{hashed_password: hashed_password},
```

File: lib/elixir_google_scraper/account/user_tokens.ex [hunk 5]
```diff
@@ -12,50 +12,41 @@ defmodule ElixirGoogleScraper.Accounts.UserToken do
...
         where: token.inserted_at > ago(@session_validity_in_days, "day"),
         select: user
+      )
 
     {:ok, query}
   end
 
-  @doc """
-  Builds a token with a hashed counter part.
+  @spec build_email_token(atom | %{:email => any, :id => any, optional(any) => any}, any) ::
```
Comment (rosle) [inline lib/elixir_google_scraper/account/user_tokens.ex]: Is this intended? 🤔

**Review by rosle — APPROVED** [review]
👍

File: lib/elixir_google_scraper/account/user_tokens.ex (hunk 5 above)
Comment (junan) [reply lib/elixir_google_scraper/account/user_tokens.ex]: My VS code plugin autogenerated it 🙈 , removed!

**Review by hanam1ni — APPROVED** [review]

**Pull request merged**